### ISSUE TITLE
Question about graphic stack 

### ISSUE THREAD

**Issue by samirhatay** [01org/XenGT-Preview-kernel#10]
Hello, 

I currently try to use XenGT in Tizen. I'm following the guide you provide with the project. My question is about the graphic stack section : Tizen uses Wayland and not Xorg. 

I am wondering if XenGT could work anyway ? 

Thanks you, 
Best regards

**Comment by jikesong** [comment]
What's your kernel (as well as gpu driver) of Tizen system? I have no idea of this OS.  However, it should be possible to setup XenGT environment with a Linux kernel based operating system.

**Comment by samirhatay** [comment]
Thanks for your reply, 

I use the kernel you provide. Also, I am working on MinnowBoard Max board which have a integrated Intel HD Graphics. 

I compiled the XenGT kernel, Xen and configured the Grub, but I have no display when I try to boot.

**Comment by jikesong** [comment]
Just googled the MinnowBoard and found it's an atom-based board? Not sure what's the microarchitecture it's based on.  If it's baytrail-based, it's possible - though may not be trivial - to enable XenGT support: we have a "byt-experiment" branch for that, but please be aware that, we don't use atom-based boards for daily development, so there may be BUGs and regressions.

**Comment by samirhatay** [comment]
According to Intel documentation about Atom SoC, baytrail is the former name to this SoC family. Should I follow the installation guide with the byt-experiment branch ? 
Thanks you

**Comment by jikesong** [comment]
@samirhatay : yes.  However, please be patient and tolerant - there may be BUGs :)
